packages/components/src/molecules/Panels/ResizableColumnsPanel.tsx

```diff
@@ -39,17 +45,57 @@ const ResizableColumnsPanel: React.FC<ResizableColumnsPanelType> = (props) => {
           onStopResize={onStopResizeCenter}
           flex={layout?.center}
```

Comment:
shouldnt be optional or if it is:  layout?.center || "center"

Reply:
the flex property is between `0` and `1` and it's show how much of the entire width it should take. Leaving it undefined will make the width take the remaining space

Reply:
No I mean shouldnt be optional  like  layout.center or we should write like layout?.center || "center"  :)  nit point tho just good to have.